### DISCUSSION TITLE
set max zoom

### DISCUSSION THREAD

**Discussion by tom-deadline** [google/model-viewer#2112] [Q&A] [answered]
is there a way to set the maximum for zooming?

**Comment by arthurrmp** [comment, marked as answer]
Yeah, you can use min-field-of-view like that

`<model-viewer ... min-field-of-view="10deg"></model-viewer>`

**Comment by tom-deadline** [comment]
@arthurrmp Thank you for your answer. I tried it and after zooming in I can't zoom out now.

**Reply by tom-deadline** [reply]
Nevermind, these settings solved it 
` min-field-of-view="90deg"
  max-field-of-view="90deg"
  camera-orbit="0deg 75deg 50%"`
